liberty-maven-plugin/src/it/server-config-props-it/src/test/java/net/wasdev/wlp/test/servlet/it/ServerConfigPropertiesTest.java

```diff
@@ -1,4 +1,4 @@
-package net.wasdev.wlp.maven.test.servlet.it;
+package net.wasdev.wlp.test.servlet.it;
```

Comment:
Did this fix cause the test to start failing? Was it a hidden failure up until now?

Reply:
Yes I think so....  I finally realized I didn't have the MAVEN_HOME set in my env..  so this test was failing...